tests/client/test_manager.py

```diff
@@ -55,3 +56,37 @@ def test_evaluations_before_running_post_fork(self, mocker):
         assert manager.splits() == []
         assert _logger.error.mock_calls == expected_msg
         _logger.reset_mock()
+
+    def test_manager_calls(self, mocker):
+        split_storage = InMemorySplitStorage()
+        split = splits.from_raw(SplitTests.raw)
+        split_storage.update([split], [], 123)
+        factory = SplitFactory(mocker.Mock(),
+            {'splits': split_storage,
+            'segments': mocker.Mock(),
+            'impressions': mocker.Mock(),
+            'events': mocker.Mock()},
+            mocker.Mock(),
+            mocker.Mock(),
+            mocker.Mock(),
+            mocker.Mock(),
+            mocker.Mock(),
+            mocker.Mock(),
+            mocker.Mock(),
+            False
+        )
+        manager = SplitManager(factory)
+        splits_view = manager.splits()
+        self._verify_split(splits_view[0])
+        assert manager.split_names() == ['some_name']
+        split_view = manager.split('some_name')
+        self._verify_split(split_view)
+
+    def _verify_split(self, split):
+        assert split.name == 'some_name'
+        assert split.traffic_type == 'user'
+        assert split.killed == False
+        assert split.treatments == ['on', 'off']
+        assert split.change_number == 123
+        assert split.configs == {'on': '{"color": "blue", "size": 13}'}
+        assert split.sets == ['set1', 'set2']
```

Comment:
Maybe a test to check that sets equals `[]` when the split has no sets could be added

Reply:
will do